modules/tinymce/src/themes/silver/main/ts/modes/Iframe.ts

```diff
@@ -148,9 +149,21 @@ const render = (editor: Editor, uiComponents: RenderUiComponents, rawUiConfig: R
     });
   }
 
+  const api: Partial<EditorUiApi> = {
+    enable: () => {
+      if (!editor.mode.isReadOnly()) {
```

Comment:
Should this be in core, rather than in the theme, as this doesn't seem like something we'd want customizable per theme? I'm wondering if we need to change core to use an facade type approach (I think that's right?) instead of an override approach. eg:

```
const renderInfo = renderThemeUi(editor);
const uiApiFacade: EditorUiApi = {
  show: Optional.from(renderInfo.api.show).getOr(Fun.noop),
  hide: Optional.from(renderInfo.api.hide).getOr(Fun.noop),
  disable: Optional.from(renderInfo.api.disable).getOr(Fun.noop),
  enable: () => {
    if (!editor.mode.isReadOnly()) {
      Optional.from(renderInfo.api.enable).map(Fun.call);
    }
  }
};
editor.ui = { ...editor.ui, ...uiApiFacade };
```
Note: We'd probably want to pull the above out into a separate function and clean up?

Reply:
Doing this would also help remove duplication between the inline/iframe code.

Reply:
Moved it to core and went with this approach